images/virtualization-artifact/pkg/controller/vmclass/internal/validators/validators_suite_test.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2024 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validators_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	. "github.com/deckhouse/virtualization-controller/pkg/controller/vmclass/internal/validators"
```

Comment:
Using the "." import is one of the recommended ways for the Ginkgo framework (as well as for Gomega). However, it is better to avoid using this for our packages